config/database.yml

```diff
@@ -29,6 +29,8 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
+  # NOTE: You will need to uncomment  and use the `username` value that corresponds with
+  # ...the POSTGRES_USER value in docker-compose.yml
```

Comment:
just to confirm -- should these be uncommented for local or docker running of the database?

Reply:
Oh yeah, I forgot to specify that. Updated with that in mind and your above comment about `docker-compose ps`

Reply:
Awesome, thanks for taking care of this again and and clarifying the instructions!